spec/services/nsm/all_adjustments_deleter_spec.rb

```diff
@@ -143,6 +153,12 @@
         expect(subject).not_to receive(:delete_disbursement_adjustments)
         subject.call
       end
+
+      it 'wont try to youth court fee adjustment if none' do
```

Comment:
nit: "wont try to remove"

Reply:
another instance of "don't copy and paste" aha